web/static/js/actions/questionnaire.js

```diff
@@ -54,7 +54,7 @@ export const CHANGE_EXPLANATION_STEP_SKIP_LOGIC = 'QUESTIONNAIRE_CHANGE_EXPLANAT
 export const CHANGE_DISPOSITION = 'QUESTIONNAIRE_CHANGE_DISPOSITION'
 export const TOGGLE_ACCEPT_REFUSALS = 'QUESTIONNAIRE_TOGGLE_ACCEPT_REFUSALS'
 export const CHANGE_REFUSAL = 'QUESTIONNAIRE_CHANGE_REFUSAL'
-export const SET_MOBILE_WEB_SMS_MESSAGE = 'QUESTIONNAIRE_SET_MOBILE_WEB_SMS_MESSAGE'
```

Comment:
why this key is removed? the web sms message is no longer used?

Reply:
We abstracted this in order to reuse code and avoid adding a key for every text in the mobile web settings. So,  `intro_message` and `sms_message` share the same `SET_MOBILE_WEB_SETTING_TEXT` key here:

https://github.com/instedd/surveda/pull/1660/commits/d346cb911fcc18aa9a3ab317203e50871e68335a#diff-41b4d8946a0a0025e042fcf90b135cd2R480

It seems OK to me, but please let me know if I'm missing something

Reply:
I see, I prefer to remove the abstraction and expand it for both fields. The rest of the codebase seems to follow that pattern. For example `setMobileWebSurveyIsOverMessage` is still present. setPrimaryColor/setSecondaryColor are not abstract. The only parametrised keys used are regarding the language which are dynamic.